test/lib/LocalizationLoader.js

```diff
@@ -453,13 +452,14 @@ describe('server/LocalizationLoader', function () {
 								'Response should have one cookie setup'
 							);
 
-							assert.strictEqual(/^locale=ru/
+							// TODO: refactoring, old value 'locale'
```

Comment:
So, if I got it right it's for cookie name parameter testing? If so, just remove the todo comment.